queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/Extensions/ChargeItemExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using fiskaltrust.ifPOS.v1;
+using fiskaltrust.Middleware.Contracts.Exceptions;
+
+namespace fiskaltrust.Middleware.Localization.QueueDEFAULT.Extensions
+{
+    public static class ChargeItemExtensions
+    {
+        public static bool IsFlagSet(this ChargeItem chargeItem)
```

Comment:
```suggestion
        public static bool IsSampleFlagSet(this ChargeItem chargeItem)
```

Maybe to clarify what this is 🤔